lib/java/README.md

```diff
@@ -42,11 +42,41 @@ The Thrift Java source is not build using the GNU tools, but rather uses
 the Gradle build system, which tends to be predominant amongst Java
 developers.
 
+Currently we use gradle 6.9.2 to build the Thrift Java source. The usual way to setup gradle
+project is to include the gradle-wrapper.jar in the project and then run the gradle wrapper to
+bootstrap setting up gradle binaries. However to avoid putting binary files into the source tree we
+have ignored the gradle wrapper files. You are expected to install it manually, as described in
+the [gradle documentation](https://docs.gradle.org/current/userguide/installation.html), or
+following this step (which is also done in the travis CI docker images):
```

Comment:
Minor punctuation, capitalization, and phrasing improvements:

```suggestion
Currently, we use Gradle 6.9.2 to build the Thrift Java source. The usual way to setup Gradle
project is to include the gradle-wrapper.jar in the project and then run the Gradle Wrapper to
bootstrap setting up Gradle binaries. However, to avoid putting binary files into the source tree, we
have excluded the Gradle Wrapper files. You are expected to install Gradle manually, as described in
the [Gradle documentation](https://docs.gradle.org/current/userguide/installation.html), or
following the following steps (which is also done in the Travis CI Docker images):
```